src/bindings.rs

```diff
@@ -0,0 +1,132 @@
+use conrod::backend::glium::glium;
+use conrod::backend::glium::glium::glutin::VirtualKeyCode;
+use serde::de;
+use std::fmt::{self, Display};
+
+/**
+ * A pressed key
+ */
+#[derive(Debug, Clone)]
+pub struct KeyCode(VirtualKeyCode);
+
+struct KeyCodeVisitor;
+
+impl<'de> de::Visitor<'de> for KeyCodeVisitor {
+    type Value = KeyCode;
+
+    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        formatter.write_str("a shortcut identifier")
+    }
+
+    fn visit_str<E>(self, value: &str) -> Result<Self::Value, E>
+    where
+        E: de::Error,
+    {
+        match value {
+            "a" => Ok(KeyCode(glium::glutin::VirtualKeyCode::A)),
```

Comment:
this isn't great, but apart from a macro I don't see a good way to fix the repetition....